es/cheatsheet-unsupervised-learning.md

```diff
@@ -1,340 +1,342 @@
 
-&#10230;
+&#10230; En un entorno de aprendizaje no supervisado, a menudo es difícil evaluar el rendimiento de un modelo ya que no contamos con las etiquetas verdaderas, como es el caso del aprendizaje supervisado.
 
 <br>
 
 **26. Silhouette coefficient ― By noting a and b the mean distance between a sample and all other points in the same class, and between a sample and all other points in the next nearest cluster, the silhouette coefficient s for a single sample is defined as follows:**
 
-&#10230;
+&#10230; Coeficiente de silueta - Sea a y b la distancia media entre una muestra y todos los demás puntos en la misma clase, y entre una muestra y todos los demás puntos en el siguiente grupo más cercano, el coeficiente de silueta s para una muestra individual se define de la siguiente manera:
 
 <br>
 
 **27. Calinski-Harabaz index ― By noting k the number of clusters, Bk and Wk the between and within-clustering dispersion matrices respectively defined as**
 
-&#10230;
+&#10230; Índice de Calinski-Harabaz - Sea k el número de conglomerados, Bk y Wk las matrices de dispersión entre y dentro de la agrupación, respectivamente, se definen como:
 
 <br>
 
 **28. the Calinski-Harabaz index s(k) indicates how well a clustering model defines its clusters, such that the higher the score, the more dense and well separated the clusters are. It is defined as follows:**
 
-&#10230;
+&#10230; El índice de Calinski-Harabaz s(k) indica qué tan bien un modelo de agrupamiento define sus grupos, de tal manera que cuanto mayor sea el puntaje, más denso y bien separados estarán los conglomerados. Se define de la siguiente manera:
 
 <br>
 
 **29. Dimension reduction**
 
-&#10230;
+&#10230; Reducción de dimensión
 
 <br>
 
 **30. Principal component analysis**
 
-&#10230;
+&#10230; Análisis de los componentes principales
 
 <br>
 
 **31. It is a dimension reduction technique that finds the variance maximizing directions onto which to project the data.**
 
-&#10230;
+&#10230; Es una técnica de reducción de dimensión que encuentra la varianza maximizando las direcciones sobre las cuales se proyectan los datos.
 
 <br>
 
 **32. Eigenvalue, eigenvector ― Given a matrix A∈Rn×n, λ is said to be an eigenvalue of A if there exists a vector z∈Rn∖{0}, called eigenvector, such that we have:**
 
-&#10230;
+&#10230; Autovalor (Eigenvalue), Autovector (Eigenvector) ― Dada una matriz A∈Rn×n, se dice que λ es un autovalor de A si existe un vector z∈Rn∖{0}, llamado autovector,  de tal manera que tenemos:
 
 <br>
 
 **33. Spectral theorem ― Let A∈Rn×n. If A is symmetric, then A is diagonalizable by a real orthogonal matrix U∈Rn×n. By noting Λ=diag(λ1,...,λn), we have:**
 
-&#10230;
+&#10230; Teorema espectral - Sea A∈Rn×n. Si A es simétrica, entonces A es diagonizable por una matriz ortogonal real U∈Rn×n. Al observar Λ=diag(λ1,...,λn), tenemos:
```

Comment:
In other translation, somebody used "_a través de una matriz ortogonal real_" instead of "por una matriz ortogonal real".